views/requests/view.erb

```diff
@@ -1 +1,33 @@
-    Requests I've received
\ No newline at end of file
+
+
+<div class= left-side-container>
+    
+        <div class="right-side-container">
+            Requests I've received
```

Comment:
Maybe use <h3> for requests received & Requests I've made

Reply:
used the h3 tag.